src/nplinker/nplinker.py

```diff
@@ -502,16 +502,16 @@ def get_common_strains(self, objects_a, objects_b, filter_no_shared=True):
 
         return common_strains
 
-    def has_bgc(self, name):
-        """Returns True if BGC ``name`` exists in the dataset"""
-        return name in self._bgc_lookup
+    def has_bgc(self, bgc_id):
+        """Returns True if BGC ``bgc_id`` exists in the dataset"""
+        return bgc_id in self._bgc_lookup
 
-    def lookup_bgc(self, name):
-        """If BGC ``name`` exists, return it. Otherwise return None"""
-        if name not in self._bgc_lookup:
+    def lookup_bgc(self, bgc_id):
+        """If BGC ``bgc_id`` exists, return it. Otherwise return None"""
+        if bgc_id not in self._bgc_lookup:
             return None
 
-        return self._bgcs[self._bgc_lookup[name]]
+        return self._bgcs[self._bgc_lookup[bgc_id]]
```

Comment:
Could be one line with ternary operator using method `has_bgc`

Reply:
Good idea! Updated for 3 lookup methods.